apps/etterlatte-saksbehandling-ui/client/src/components/behandling/sidemeny/SettPaaVent.tsx

```diff
@@ -11,30 +11,48 @@ import { isPending } from '~shared/api/apiUtils'
 import { erOppgaveRedigerbar, OppgaveDTO } from '~shared/types/oppgave'
 import { useAppDispatch } from '~store/Store'
 import { settOppgave } from '~store/reducers/OppgaveReducer'
+import { Text } from '~components/behandling/attestering/styled'
 
 interface Props {
   oppgave: OppgaveDTO | null
   redigerbar: boolean
 }
 
+enum PaaventAarsak {
+  OPPLYSNING_FRA_BRUKER = 'Opplysning fra bruker',
+  OPPLYSNING_FRA_ANDRE = 'Opplysning fra andre',
+  KRAVGRUNNLAG_SPERRET = 'Kravgrunnlag sperret',
+  ANNET = 'Annet',
+}
+
 export const SettPaaVent = ({ oppgave, redigerbar }: Props) => {
   if (!oppgave || !erOppgaveRedigerbar(oppgave?.status)) return null
 
   const dispatch = useAppDispatch()
+  type settPaaVentTyper = Array<keyof typeof PaaventAarsak>
 
   const [frist, setFrist] = useState<string>(oppgave.frist)
   const [merknad, setMerknad] = useState<string>(oppgave.merknad || '')
   const [settPaaVent, setVisPaaVent] = useState(false)
+  const [aarsak, setAarsak] = useState<settPaaVentTyper[number]>()
+  const [aarsakError, setAarsakError] = useState<boolean>(false)
```

Comment:
Synes typebruken her er mildt sagt tung å lese.  Eks. `settPaaVentTyper[number]` hvor man må hoppe rundt i koden for å skjønne hva i all verden typen faktisk er

Reply:
Endret til kun å bruke enumen og ikke via number